### PR TITLE
Fixes for Windows GCC interop needed by RPy2 and (potentially) CVXOPT

### PR DESCRIPTION
**Pull request by mingwandroid**

### CODE REVIEW

**Comment by mingwandroid** [comment]
Here you go @ocepaf

**Comment by jjhelmus** [comment]
LGTM.  Wanted to check if these new patches should be applied on all platforms or just Windows?

**Comment by mingwandroid** [comment]
All patches on all platforms always.

**Comment by jjhelmus** [comment]
Thanks for the updates @mingwandroid

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,9 +19,11 @@ source:
     # - https://bugs.python.org/issue28708
     # - https://github.com/zeromq/libzmq/issues/1165
     - fd_setsize.patch                # [win]
+    - mingw-w64--DMS_WINXX.patch
+    - win-msc_ver-1900.patch
 
 build:
-  number: 1
+  number: 2
```
Comment (isuruf) [inline recipe/meta.yaml]: Github is playing with us. This was `2` before and there should be no change here. Build number `2` was set in this PR https://github.com/conda-forge/python-feedstock/pull/139/files.

cc @ocefpaf

Comment (ocefpaf) [inline recipe/meta.yaml]: Thanks @isuruf! That explains the `rpy2` problem. I'll send a PR to bump it now.

**Pull request merged**